src/components/download/helperFunctions.js

```diff
@@ -623,15 +623,19 @@ export const entropyTSV = (dispatch, filePrefix, entropy) => {
 /**
  * Write out Auspice JSON(s) for the current view. We do this by re-fetching the original
  * JSON because we don't keep a copy of the unprocessed data around.
- * 
+ *
  * Sidecar files are not fetched, but we can also download them if desired.
- * 
+ *
  * Note that we are not viewing a narrative, as the download button functionality is disabled
  * for narratives.
  */
-export const auspiceJSON = (dispatch) => {
+export const auspiceJSON = (dispatch, datasetNames) => {
   const filenames = [];
-  for (const datasetName of getDatasetNamesFromUrl(window.location.pathname)) {
+  if (!datasetNames.every(Boolean)) {
+    console.error(`Unable to fetch empty dataset names: ${JSON.stringify(datasetNames)}`);
+    return dispatch(errorNotification({message: "Unable to download Auspice JSON (see console for more info)"}))
```

Comment:
`datasetNames` is always an array of length 2. So for `/zika` it's `[ "zika", undefined ]`, and thus we end up showing a error notification and not downloading the JSON. 

Perhaps the intention here was `.all(Boolean)`? But that's unnecessary because of the conditions under which `auspiceJSON()` gets called.

Reply:
Ah, thanks for catching that. I meant `.some(Boolean)` to verify at least one of them is truthy. 

> But that's unnecessary because of the conditions under which auspiceJSON() gets called.

Yeah, I added this guard to get error notifications on unexpected use of `auspiceJSON()` instead of just letting it hang on the "Preparing Auspice JSON..." notification.

Reply:
Fixed up in [force-push](https://github.com/nextstrain/auspice/compare/45bd85d4cca351ab5447c02d507b8dba69cdce05..dbfce230392927f83a921c29e8447287d4133187)

Reply:
> Ah, thanks for catching that. I meant `.some(Boolean)` to verify at least one of them is truthy.

Nice one. I don't know what I meant by suggesting `.all`, which isn't even an array prototype method!